roll.go

```diff
@@ -26,23 +26,9 @@ func main() {
 	appengine.Main()
 }
 func handle(w http.ResponseWriter, r *http.Request) {
-	//numberOfDice, sides:=parseDice(r.URL.Path)
-	content := r.URL.Path
-	if !diceRegexp.MatchString(content) {
-		fmt.Fprintf(w, "%s is not a valid roll\n", strings.Replace(content, "/", "", -1))
-		return
-	}
-	numberOfDice, _ := strconv.ParseInt(diceRegexp.FindStringSubmatch(content)[1], 10, 0)
-	sides, _ := strconv.ParseInt(diceRegexp.FindStringSubmatch(content)[2], 10, 0)
-	rollResult := roll(int(numberOfDice), int(sides))
-	diceWord := ""
-	if int(numberOfDice) > 1 {
-		diceWord = "dice"
-	} else {
-		diceWord = "die"
-	}
-
-	fmt.Fprintf(w, "You rolled %d on %d %d sided %s\n", rollResult, numberOfDice, sides, diceWord)

```

Comment:
I removed this functionality and just returned the number. This is probably not ideal, but I wanted to focus on just the parsing and evaluating logic.